curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698440c54c8823cc60af8bc.md

```diff
@@ -46,10 +47,10 @@ assert.deepStrictEqual(
 The three `li` elements should be located between the `ul` element's opening and closing tags.
 
 ```js
-assert(
+assert.lengthOf(
```

Comment:
please check all cases in which you use `lengthOf`, then there don't use `length`